charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.9.2]
+### Added
+- Added custom opensearch deployment annotation through values.yaml 
```

Comment:
[NIT] Thanks @prathaptce noticed one more, please check
- Added custom opensearch and dashboard annotations through values.yaml.

Reply:
thanks. updated the same